corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -160,10 +161,19 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
                 endPage = currentPage + maxPagesAfterCurrentPage;
             }
         }
+
+        const rowRange = [5, 10, 25, 50, 100],
+            hasMultiplePages = totalPages > 1,
+            hasAtLeastMinimumItemsPerPage = totalItems > rowRange[0],
```

Comment:
when would `hasAtLeastMinimumItemsPerPage` be true but `hasMultiplePages` not?

Reply:
Say the user has selected 10 items per page, and there are 8 items: `totalPages` would be 1, but `totalItems` is greater than the bare minimum (5).

Reply:
But why would you have to show pagination in that case? wouldn't everything be on the one page?

Reply:
It gives the user the option to decrease the number of items per page - they'll only see that dropdown, there are already other checks that hide the buttons to jump to different pages if there's only one page.